lib/api/list.js

```diff
@@ -63,6 +75,17 @@ List.prototype = {
       _method: 'GET'
     });
     return this._connection.post(path, {data: options});
+  },
+
+  isMember: function(listId, leadIds) {
+    var path = util.createPath('lists', listId, 'leads', 'ismember.json');
+    leadIds = [].concat(leadIds); // ensure it's an array
+    return this._connection.get(path + "?id=" + leadIds.join('&id='));
```

Comment:
The `get` call accepts a second parameter for the query parameters to make it more consistent with the rest of the code.

```js
this._connection.get(path, {id: leadIds.join('&id=')})
```